src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -16,117 +16,60 @@
  */
 package com.g4s8.ghman.bot;
 
-import com.g4s8.ghman.env.EnvironmentVariables;
-import com.g4s8.ghman.user.Users;
-import com.g4s8.teletakes.bot.BotSimple;
-import com.g4s8.teletakes.fk.FkCallbackQuery;
-import com.g4s8.teletakes.fk.FkCommand;
-import com.g4s8.teletakes.tk.TkFallback;
-import com.g4s8.teletakes.tk.TkFork;
 import com.jcabi.aspects.Tv;
 import com.jcabi.log.Logger;
-import java.util.regex.Pattern;
 import org.telegram.telegrambots.ApiContextInitializer;
 import org.telegram.telegrambots.TelegramBotsApi;
 import org.telegram.telegrambots.exceptions.TelegramApiRequestException;
 import org.telegram.telegrambots.generics.BotSession;
+import org.telegram.telegrambots.generics.LongPollingBot;
 
 /**
  * Telegram bot entry point.
  *
  * @since 1.0
- * @todo #2:30mins Fix the ClassDataAbstractionCouplingCheck & PMD rule
- *  exclusions in BotApp class
  * @todo #11:30mins Patterns in callback queries chains are distributed in
  *  multiple classes: for example, pattern for close issue is given here and in
  *  TkCloseIssue. Find a way to avoid this by maybe having string pattern
  *  representation here and passing Matcher to telegram takes (TkCloseIssue) or
  *  by having one class implementing Fork and containing both the current code
  *  of TkCloseIssue and the creation of the FkCallbackQuery.
- * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@victornoel hm, this todo is referencing  to the part of the class you moved into `GhBot`, can we do smth about it? Correct or move todo or at least add comment to the issue manually?

Reply:
@olenagerasimova good point, for the record it can be moved as long as the text is not modified, 0pdd will recognize it as the same